ctac_to_mu_values problem if BUILD_EXECUTABLES=OFF

error is
```
CMake Error at src/utilities/CMakeLists.txt:90 (target_link_libraries):
  Cannot specify link libraries for target "ctac_to_mu_values" which is not
  built by this project.
```

--- a/src/utilities/CMakeLists.txt
+++ b/src/utilities/CMakeLists.txt
@@ -86,6 +86,6 @@ endif()
 
 include(stir_exe_targets)
 
-if (nlohmann_json_FOUND)
+if (BUILD_EXECUTABLES AND nlohmann_json_FOUND)
   target_link_libraries(ctac_to_mu_values nlohmann_json::nlohmann_json)
 endif()

